sections/docker.zsh

```diff
@@ -23,6 +23,11 @@ SPACESHIP_DOCKER_VERBOSE="${SPACESHIP_DOCKER_VERBOSE=false}"
 spaceship_docker() {
   [[ $SPACESHIP_DOCKER_SHOW == false ]] && return
 
+  # Check if podman is used instead of docker
+  spaceship::exists podman || return 
```

Comment:
Please rephrase this. If `podman` doesn't exist, the rest of the code won't be executed, including all Docker related code.

I think it would be better to create a separate podman section based on the docker section code instead of mixing these two:

- Docker has a lot of more options that podman doesn't provide by design (like `docker-compose`, remote `DOCKER_HOST` handling or Docker contexts)
- Maintenance for both sections will be easier, especially when adding runtime specific features

Reply:
Okay i don't have time for the coming 1,5 week but i can break it out a create a nr PR during the holidays.